.github/actions/test/action.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Test'
+description: 'Code qa and tests'
+inputs:
+  aws-role-arn:
+    description: 'ARN of AWS account to assume'
+    required: true
+  codecov-token:
+    description: 'Token for uploading to codecov.io'
+    required: true
+
+
+runs:
+  using: composite
+  steps:
```

Comment:
One more small thing
Somewhere in here, can we run

`uv lock --locked`

That will fail the build if there are changes to `pyproject.toml` but we've forgotten to rebuild the lockfile